dts/arm/yaml/partition.yaml

```diff
@@ -0,0 +1,43 @@
+---
+title: Flash Partitions
+id: fixed-partition
+version: 0.1
+
+description: >
+    This binding gives a base FLASH partition description
+
+properties:
+  - compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "fixed-partitions"
+
+  - partition\d+:
+      type: string
+      generation: define
+
+      properties:
+        - reg:
+            type: array
+            description: mmio register space
```

Comment:
these seems wrong, while is mmio space, saying its register space seems a little misleading. (I assume we are talking about flashes that are MMIO direct access)